BAZProjectC1/Components/BasicInfoMovie/BasicInfoMovie.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  BasicInfoMovie.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 23/09/22.
+//
+
+import UIKit
+
+class BasicInfoMovie: UIView {
+    private var viewBackgroundColor: UIColor? = .clear
+    private lazy var backView: UIView = {
+        let backView = UIView(frame: self.bounds)
+        backView.layer.cornerRadius = RounderBorderStyleForView.rounded.rawValue
+        return backView
+    }()
+    
+    internal convenience init(frame:CGRect,viewBackgroundColor: UIColor) {
```

Comment:
`internal` es el default, no es necesario escribirlo explícitamente.

También cuida los espacios entre los parámetros de la función

```suggestion
   convenience init(frame: CGRect, viewBackgroundColor: UIColor) {
```